data/json/items/gun/9mm.json

```diff
@@ -115,7 +115,7 @@
     "durability": 6,
     "magazine_well": 1,
     "magazines": [
-      [ "9mm", [ "glockmag", "glockbigmag" ] ],
+      [ "9mm", [ "glockmag", "glockbigmag", "glock_drum_50rd", "glock_drum_100rd" ] ],
```

Comment:
Would the 17-round mag not fix in the Glock 19? Isn't the 19 just a compact version of the 17?

Reply:
I'll have to check that.